test/e2e/operator_test.go

```diff
@@ -506,7 +506,7 @@ func TestDNSOverTLSForwarding(t *testing.T) {
 	// Create a separate namespace for the upstream resolver. Deleting this namespace will clean up the resources created.
 	tlsUpstreamNamespace := &corev1.Namespace{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: tlsUpstreamName,
+			Name: "openshift-testdnsovertls",
```

Comment:
What's the purpose of changing the namespace name here?  Does it still clean up the resources that were created?

Reply:
Namespaces prefaced with "openshift" are less restricted.